src/ildasm/dasm.cpp

```diff
@@ -1455,7 +1498,7 @@ mdToken ResolveReflectionNotation(BYTE* dataPtr,
         {
             *pch = 0;
             for(szAssembly = pch+1; *szAssembly == ' '; szAssembly++);
-            pch = strchr(szAssembly,',');
+            pch = (char*)strchr(szAssembly,',');
             if(pch) *pch = 0;
```

Comment:
The szAssembly is const char* and you cast the constness away and write to it. Can you remove the const from the szAssembly then since it is misleading?